content/3_Silver/Flood_Fill.mdx

```diff
@@ -372,117 +466,175 @@ static void floodfill(int r, int c, int color){
 
 ### Implementation
 
+<Warning>
+
+Recursive implementations of floodfill can sometimes lead to a stack overflow.
+To avoid that, we can use a stack data structure in an iterative implementation
+as follows:
+
+</Warning>
+
 <LanguageSection>
 
 <CPPSection>
 
 ```cpp
-#include <bits/stdc++.h>
-using namespace std;
-
-#define FOR(i,a,b) for (int i = (a); i < (b); ++i)
-#define F0R(i,a) FOR(i,0,a)
+#include <iostream>
+#include <string>
```

Comment:
i don't think you need this? (`#include <string>`)

Reply:
It's needed for the representation of each row.